src/billing/BillingAccount/BillingAccountSummary.tsx

```diff
@@ -0,0 +1,55 @@
+import { Icon, Link } from '@terra-ui-packages/components';
+import React, { ReactNode } from 'react';
+import { BillingAccountStatus, getBillingAccountIconProps } from 'src/billing/utils';
+import colors from 'src/libs/colors';
+import { contactUsActive } from 'src/libs/state';
+import { topBarHeight } from 'src/libs/style';
+
+interface BillingAccountSummaryProps {
+  done: number;
+  error: number;
+  updating: number;
+}
+
+export const BillingAccountSummary = (props: BillingAccountSummaryProps): ReactNode => {
```

Comment:
This is the component. I had to hack state locally to see it.

<img width="526" alt="image" src="https://github.com/user-attachments/assets/718b9fd6-f6e2-4bc1-bc9d-91e02e6e757d">